util/kustomize/kustomize.go

```diff
@@ -448,7 +448,12 @@ func getSemverSafe() *semver.Version {
 }
 
 func Version() (string, error) {
-	cmd := exec.Command("kustomize", "version", "--short")
+	return versionWithBinaryPath(&kustomize{})
+}
+
+func versionWithBinaryPath(k *kustomize) (string, error) {
+	executable := k.getBinaryPath()
+	cmd := exec.Command(executable, "version", "--short")
```

Comment:
`--short` has been deprecated, should we move it? It also makes parsing the version way easier now.

Reply:
@rumstead I understand your point, but this is outside the scope of this PR. It should be addressed in a separate PR.